### DISCUSSION TITLE
[BUG] Submit-PnPSearchQuery fails when using the "-All" parameter

### DISCUSSION THREAD

**Discussion by ktlayton** [pnp/powershell#3998] [Q&A]
All of a sudden when making a Submit-PnPSearchQuery with the "-All" parameter it throws the following error:

We didn't understand your search terms. Make sure they're using proper syntax.

Here is my module information, and the operating system is Windows

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Manifest   1.12.0     PnP.PowerShell                      {Add-PnPAdaptiveScopeProperty, Add-PnPPropertyBagValue, Add-PnPSiteClassification, Copy-PnPFolder...}
Manifest   1.10.0     PnP.PowerShell                      {Add-PnPAdaptiveScopeProperty, Add-PnPPropertyBagValue, Copy-PnPFolder, Export-PnPClientSidePage...}

**Comment by themacksta** [comment]
We are seeing similar issue, we are using SP API's direct though. Don't think it's a PNP issue, looks like it's SP API. Have lodged a MS support request 
**Case Details** ‎Search API Failing‎ (#‎2406040030001324‎)

This is the API endpoint we are hitting (just in case we have a MS engineer lurking here)
POST
https://xxx.sharepoint.com/sites/xxx/_api/search/postquery
**Request**
<hidden, has sensitive info>
**Response**
{
    "odata.error": {
        "code": "-1, Microsoft.Office.Server.Search.REST.SearchServiceException",
        "message": {
            "lang": "en-US",
            "value": "We didn't understand your search terms. Make sure they're using proper syntax."
        }
    }
}
